### PR TITLE
Welling/unified assay names 202011 2

### PR DESCRIPTION
**Pull request by jswelling**
Added vitessce-hints to assay type schema; updated assay_types.yaml accordingly.

### CODE REVIEW

**Comment by yuanzhou** [comment]
I'll merge the deploy on DEV to test once @ngehlenborg approves this PR.

**Review by ngehlenborg — APPROVED** [review]
Sorry for dragging my feet on this!

**Pull request merged**
